back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/taggings_controller.rb

```diff
@@ -35,6 +36,19 @@ def destroy
           end
         end
 
+        def bulk_create
```

Comment:
I'm not sure if we should do more in terms of error handling here - like returning unprocessable entry error is tag_id or filters is missing from the request? Couldn't find a nice example in the codebase on how to do that in this case.

Reply:
We still get an error - just a different one so I guess it's kind of ok?

Reply:
The only reason why a Tagging record can realistically be invalid is because there is already a duplicate for a given input/tag combination. That's something we expect and accept, so I think it's fine like this.

Error handling in bulk_create is always difficult, but in this case it's such a simple model that we should be fine. I wouldn't even do the `puts` call, makes the logs dirty with no real benefit imo, we expect this to happen frequently and that's fine.